common/perception_utils/include/perception_utils/conversion.hpp

```diff
@@ -12,25 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "perception_utils/perception_utils.hpp"
+#ifndef PERCEPTION_UTILS__CONVERSION_HPP_
+#define PERCEPTION_UTILS__CONVERSION_HPP_
+
+#include "autoware_auto_perception_msgs/msg/detected_objects.hpp"
+#include "autoware_auto_perception_msgs/msg/tracked_objects.hpp"
 
 namespace perception_utils
 {
-std::uint8_t getHighestProbLabel(
-  const std::vector<autoware_auto_perception_msgs::msg::ObjectClassification> & classification)
-{
-  std::uint8_t label = autoware_auto_perception_msgs::msg::ObjectClassification::UNKNOWN;
-  float highest_prob = 0.0;
-  for (const auto & _class : classification) {
-    if (highest_prob < _class.probability) {
-      highest_prob = _class.probability;
-      label = _class.label;
-    }
-  }
-  return label;
-}
-
-autoware_auto_perception_msgs::msg::DetectedObject toDetectedObject(
+inline autoware_auto_perception_msgs::msg::DetectedObject toDetectedObject(
```

Comment:
By the way, why did you move functions to `.hpp`?
I think it's not recommended, it's just allowed for work time efficiency.
So you don't need to move all functions to header files.

Reply:
Thank you for thoughtful review.
I intend to imitate to tier4_autoware_util composition, but I agree with your recomendation.
So I fixed at [c2f9caf](https://github.com/autowarefoundation/autoware.universe/pull/1954/commits/c2f9cafae0a58f8391d7e42aa5debebd17dd0f22).

Reply:
Regarding `tier4_autoware_util`, I've shortened the process previously because it was in the prototyping phase.
I'm sorry for confusing you.

And thank you for fixing it.